bind/flag.go

```diff
@@ -93,6 +93,12 @@ func HTTPProxyConfig(fs *pflag.FlagSet, cfg *forwarder.HTTPProxyConfig, lcfg *lo
 	HTTPServerConfig(fs, &cfg.HTTPServerConfig, "", forwarder.HTTPScheme, forwarder.HTTPSScheme)
 	LogConfig(fs, lcfg)
 
+	fs.StringSliceVar(&cfg.ExtraAddresses, "extra-address", cfg.ExtraAddresses,
+		"<host:port,...>"+
+			"Additional addresses to listen on. "+
+			"The flag can be specified multiple times to add multiple addresses. "+
+			"Bind failure is not critical, the proxy will continue to run. ")
```

Comment:
can we make it configurable? In my request, bind failures shouldn't be critical (and it should be a default) but it feels like it should be more generic

Reply:
We can always make it configurable. The question is do we need it right now? If not, I'd opt for not adding it, to keep the flags simple.

Reply:
fair point. Configurability is not needed right now and this functionality is not battle-tested yet, let's see how it goes.